### ISSUE TITLE
Proper implementation of the back pressure protocol

### ISSUE THREAD

**Issue by cescoffier** [quarkusio/quarkus#9329]
Look at the gRPC backpressure protocol and provide integration for Mutiny.

**Comment by michalszynkiewicz** [comment]
@cescoffier could you add some pointers for this task?

**Comment by cescoffier** [comment]
The best explanation I can found is there: https://github.com/salesforce/reactive-grpc#understanding-reactive-grpc-flow-control.

Basically, we need to bridge the gRPC flow control mechanism with the reactive streams flow control used by Mutiny.

**Comment by weversiep** [comment]
Hello, just a question...  do you know when the backpressure implementation will become available? We are really looking forward to using it in our applications.

**Comment by michalszynkiewicz** [comment]
@weversiep it's still on my roadmap but I didn't plan to go back it very soon. Did you encounter any problems with the current implementation?

**Comment by weversiep** [comment]
@michalszynkiewicz in our microservices landscape we have a Quarkus/gRPC eventing service that handles Kafka connectivity. 
The other services can work with Kafka messages that the eventing service pushes onto a gRPC-stream.
The receiving services need to implement pushback, otherwise they might get overloaded, but currently I see no way to do this using Quarkus.
So I think this is the bridge for gRPC flow control / Mutiny that @cescoffier mentioned.

**Comment by cescoffier** [comment]
Exactly, right now there are 2 different protocols, not connected. This issue is about handling that properly.

**Comment by weversiep** [comment]
Allright, thank you for your response.

**Comment by cescoffier** [comment]
Will be part of #24754

**Comment by cescoffier** [comment]
Fied as part of https://github.com/quarkusio/quarkus/issues/24754
